### ISSUE TITLE
Seek causes video to stop playing

### ISSUE THREAD

**Issue by vanice** [Microsoft/FFmpegInterop#55]
Is there any update on the Seek issues described in several other reports? I've tried the Seek Completed event work around and it still screws up when i change the position via code like Postition += Position + TimeSpan.FromSeconds(30);

BTW, i love this capability and its awesome for us UWP devs but so far this doesn't seem to be stable enough for production use.

**Comment by mitchcapper** [comment]
Seeking does seem broken on HLS streams.

**Comment by mitchcapper** [comment]
https://github.com/reego-fr/FFmpegInterop fixes this a bit (so trying to seek will work or at least jump to the right position) it then jumps to the end of the recording so not fixed. Going to talk more on PR #47.

**Comment by iamstudentVN** [comment]
@mitchcapper 
hi! i have this flv video https://www.youtube.com/watch?v=m7PZSUsGQKU 
It can't seek

**Comment by reego-fr** [comment]
Hi,
to summarize what is said in other posts, this is probably caused by a bug in the media stream source. It mostly happens when playing uncompressed video samples but I think it sometimes happens with compressed samples. I might be wrong but I think it is worth investigating. 
Disabling the media stream source buffer apparently fixes it but other issues may arise because of the lack of a buffer in some situations.

**Comment by iamstudentVN** [comment]
Yeah, win10 mobile was dead. this is not necessary any more. Thanks everyone!
